cmd/handlers.c

```diff
@@ -47,6 +53,14 @@ const CmdList commands[] = {
   {CMD_REST_REQUEST,    "REST_REQ",       REST_Request},
   {CMD_REST_SETHEADER,  "REST_SETHDR",    REST_SetHeader},
 #endif
+#ifdef TCP
+  {CMD_TCP_SETUP,      "TCP_SETUP",     TCP_Setup},
+  {CMD_TCP_SEND,       "TCP_SEND",      TCP_Send},
```

Comment:
I followed what is written in the rest.c comment line 40
```
They never call any 'free' and given that the attached MCU could restart at
// any time, we cannot really rely on the attached MCU to call 'free' ever, so better do without.
```
Instead of close, I would rather add a disconnect function to let the uC decide when to disconnect from a server, but then again, the socket connection might stay open when the uC restarts and a reconnect might fail.